app/services/user/create.rb

```diff
@@ -0,0 +1,23 @@
+class User::Create < ApplicationService
+  attr_accessor :user
+
+  def initialize(params)
+    @user = params[:user]
```

Comment:
I would name it `@params` because '@user` is confusing that it is already an object